communicator/StartPage.qml

```diff
@@ -0,0 +1,87 @@
+import QtQuick
+import QtQuick.Controls
+import QtQuick.Layouts
+import QtQuick.Window
+Item {
+    ColumnLayout {
+        anchors.fill: parent
+        spacing: 10
+
+        Text {
+            text: "Simple Text Communicator"
+            font.pixelSize: 40
+            color: "black"
+            // anchors.horizontalCenter: parent.horizontalCenter
+            Layout.alignment: Qt.AlignHCenter
+        }
+
+        //@TODO add logo here
+        // Item {
+        //     Layout.alignment: Qt.AlignHCenter
+        //     width: parent.width / 2
+        //     height: parent.height / 4
+
+        //     Image {
+        //         anchors.top: parent.bottom
+        //         source: "logo.png"
+        //         width: parent.width / 2
+        //         height: parent.height / 4
+        //     }
+        // }
+
+        RowLayout {
+            anchors {
+                horizontalCenter: parent.horizontalCenter
+                bottom: parent.bottom
+
+            }
```

Comment:
When I run it I've got a message from qt "QML RowLayout: Detected anchors on an item that is managed by a layout. This is undefined behavior; use Layout.alignment instead.", please check it :)